mods/cnc/languages/nod06abc/en.ftl

```diff
@@ -0,0 +1,4 @@
+steal-nuclear-detonator = Steal the GDI nuclear detonator.
+destroy-gdi-supporter-houses = Destroy the houses of the GDI supporters
+    in the village.
```

Comment:
```suggestion
destroy-gdi-supporter-houses = Destroy the houses of the GDI supporters\nin the village.
```
?

Reply:
No.

Reply:
Ok, then `investigate-village` will need the `\n` removed.